js/views/userShort.js

```diff
@@ -12,31 +12,29 @@ export default class extends BaseVw {
     this.guid = options.guid || this.model.get('guid');
     this.ownGuid = this.guid === app.profile.id;
     this.profileArgs = {}; // create blank for placeholder render
-    this.followedByYou = app.ownFollowing.get(this.guid) !== undefined;
+    this.followedByYou = followedByYou(this.guid);
     // TODO: add in code to determine if this user is in the viewer's moderator list
     this.ownMod = false;
-
-    this.loadUser();
-    /* the view should be rendered when it is created and before it has data, so it can occupy
-       space in the DOM while the data is being fetched. */
+    this.fetched = false;
+    this.loading = true;
   }
 
   loadUser(guid = this.guid) {
     let profile;
-    let profileFetch;
+    this.fetched = true;
 
-    this.loading = true;
+    if (this.isRemoved()) return;
```

Comment:
This line should be instead both the done() and fail() handlers of the profileFetch.